### PR TITLE
Add documentation for Hydrawise component

### PR DESCRIPTION
**Pull request by ptcryan**
**Description:**
Documentation for the Hydrawise component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14055

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/binary_sensor.hydrawise.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+title: "Hunter Hydrawise Binary Sensor"
+description: "Instructions on how to integrate your Hunter Hydrawise Wi-Fi irrigation control system within Home Assistant."
+date: 2018-04-11 08:02
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hydrawise_logo.png
+ha_category: Binary Sensor
+ha_release: 0.68
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.hydrawise.markdown]: 0.71

File: source/_components/binary_sensor.hydrawise.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Once you have enabled the `hydrawise` component, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+binary_sensor:
+  - platform: hydrawise
+```
+
+{% configuration %}
+
```
Comment (MartinHjelmare) [inline source/_components/binary_sensor.hydrawise.markdown]: Remove blank line.

File: source/_components/hydrawise.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+title: "Hunter Hydrawise"
+description: "Instructions on how to integrate your Hunter Hydrawise Wi-Fi irrigation control system within Home Assistant."
+date: 2018-04-11 08:02
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hydrawise_logo.png
+ha_category: Hub
+ha_release: 0.68
```
Comment (MartinHjelmare) [inline source/_components/hydrawise.markdown]: 0.71

File: source/_components/sensor.hydrawise.markdown [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+title: "Hunter Hydrawise Sensor"
+description: "Instructions on how to integrate your Hunter Hydrawise Wi-Fi irrigation control system within Home Assistant."
+date: 2018-04-11 08:02
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hydrawise_logo.png
+ha_category: Sensor
+ha_release: 0.68
```
Comment (MartinHjelmare) [inline source/_components/sensor.hydrawise.markdown]: 0.71

File: source/_components/switch.hydrawise.markdown [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+title: "Hunter Hydrawise Switch"
+description: "Instructions on how to integrate your Hunter Hydrawise Wi-Fi irrigation control system within Home Assistant."
+date: 2018-04-11 08:02
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: hydrawise_logo.png
+ha_category: Switch
+ha_release: 0.68
```
Comment (MartinHjelmare) [inline source/_components/switch.hydrawise.markdown]: 0.71

**Comment by ptcryan** [comment]
HA release information was updated. Thanks.

**Review by MartinHjelmare — APPROVED** [review]
:tada:

**Pull request merged**